pandas/core/ops.py

```diff
@@ -186,8 +186,10 @@ def add_special_arithmetic_methods(cls, arith_method=None,
     arith_method : function (optional)
```

Comment:
it does not make sense for arith_method to be an optional argument because _create_methods will try to call it without checking if it is None